contracts/AMM/Referrals.sol

```diff
@@ -59,6 +80,43 @@ contract Referrals is Initializable, ProxyOwned, ProxyPausable, ProxyReentrancyG
         }
     }
 
+    /// @notice set Referral fees
+    /// @param _referrerFeeDefault how much of a fee to pay to referrers
+    /// @param _referrerFeeSilver how much of a fee to pay to silver referrers
+    /// @param _referrerFeeGold how much of a fee to pay to gold referrers
+    function setReferrerFees(
+        uint _referrerFeeDefault,
+        uint _referrerFeeSilver,
+        uint _referrerFeeGold
+    ) external onlyOwner {
+        uint TWO_PERCENT = 0.02 * 1e18;
+        require(
+            _referrerFeeDefault < TWO_PERCENT && _referrerFeeSilver < TWO_PERCENT && _referrerFeeGold < TWO_PERCENT,
+            "Maximum referrer fee is 2%"
```

Comment:
I think its best to say "Maximum referrer fee exceeded" as we might change it from 2% at some point. Generally not ideal to hardcode values.